Rakefile.rb

```diff
@@ -23,9 +29,12 @@
     sh "cd #{PROJECT_DIR} && bundle update"
     puts "building integration #{ENV['INTEGRATION']}"
 
+    manifest = JSON.parse(File.read("/#{ENV['INTEGRATIONS_REPO']}/#{ENV['INTEGRATION']}/manifest.json"))
+    integration_version = manifest['version'] || ENV['VERSION']
```

Comment:
After an offline discussion, removing `ENV['VERSION']` is the best way to avoid confusion while still having a readable code. Every manifest should have a version, and every integration should have a manifest.